pkg/kubelet/kubelet.go

```diff
@@ -2473,16 +2465,8 @@ func (kl *Kubelet) fastStatusUpdateOnce() {
 				}
 			}
 		}
-		if len(node.Spec.PodCIDRs) != 0 {
-			podCIDRs := strings.Join(node.Spec.PodCIDRs, ",")
-			if _, err := kl.updatePodCIDR(podCIDRs); err != nil {
-				klog.ErrorS(err, "Pod CIDR update failed", "CIDR", podCIDRs)
-				continue
-			}
-			kl.updateRuntimeUp()
-			kl.syncNodeStatus()
-			delay = 2 * baseDelay
-		}
```

Comment:
I think this is correct, https://github.com/kubernetes/kubernetes/issues/62288, we should not optimize this, since seems it was implementation specific